test/fish/non-root-user.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Optional: Import test library bundled with the devcontainer CLI
+source dev-container-features-test-lib
+
+# Feature-specific tests
+check "fisher" su vscode -c "fisher -v"
```

Comment:
This user is not guaranteed to be present in the base image. Sometimes it's `node` or something else entirely.

We need to cover those cases.

I believe that upstream there is a battle-tested way to check which user is present: https://github.com/devcontainers/features/blob/ab021f0dfb0b4d28a2a71fd4b8502f5cb0b72004/src/node/install.sh#L38

Reply:
Since this test is running only on `mcr.microsoft.com/devcontainers/base:debian`, it is known that the non-root user is `vscode`.
(It is possible that destructive changes will be made in the future devcontainers/images#145, but I don't expect this to change, at least not for some time to come.)

Nevertheless, it seems better to include the part you pointed out in test.sh and remove the scenario test.

Reply:
Ah, I see.

> Nevertheless, it seems better to include the part you pointed out in `test.sh` and remove the scenario test.

I agree. It makes sense to run all tests for the same base images.